styles/base.css

```diff
@@ -1,4 +1,22 @@
 h1 {
-  font-size: 30px;
+  font-family: 'Share Tech', sans-serif;
+  /*font-size: 30px;*/
   margin-top: 0;
 }
+
+.project-picture img {
+  width: 400px;
+  height: 280px;
+}
+
+h2{
+  font-family: 'Share Tech', sans-serif;
```

Comment:
You shouldn't need to repeat the font-family in each h tag unless you are using a different font-family.